include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -177,6 +298,23 @@ class SplineEvaluator
         return eval_no_bc<eval_deriv_type>(coord_eval, spline_coef);
     }
 
+    /**
+     * @brief Derivate spline function (described by its spline coefficients) on a mesh.
```

Comment:
Recurring error in this file:
![image](https://github.com/CExA-project/ddc/assets/25363473/62e61df3-046d-4ed6-9a14-883b3423decf)

The verb you are looking for is differentiate
![image](https://github.com/CExA-project/ddc/assets/25363473/c081a717-596b-4099-80cc-46551ea54fea)

The noun is indeed derivative though